Source/FullScreenPokemon.ts

```diff
@@ -9103,9 +9174,10 @@ module FullScreenPokemon {
          * @param location   The name of the Location within the Map.
          * @param noEntrance    Whether or not an entry Function should
          *                      be skipped (by default, false).
+         * @param flying   Whether or not the Player is flying to a new location.
          * @remarks Most of the work here is done by setLocation.
          */
-        setMap(name: string, location?: string, noEntrance?: boolean): void {
+        setMap(name: string, location?: string, noEntrance?: boolean, flying?: boolean): void {
```

Comment:
Adding a `flying?: boolean` parameter isn't the right way to do it. What if future concepts like Dig or Dive need to be implemented?